hkmc2/shared/src/test/mlscript-compile/Predef.mls

```diff
@@ -31,6 +31,15 @@ fun checkArgs(functionName, expected, isUB, got) =
     //   + " arguments but got " + got)
   else ()
 
+module TraceLogger with
+  mut val indentLvl = 0
+  fun indent() =
+    let prev = indentLvl
+    set indentLvl = indentLvl + 1
```

Comment:
Again, why not use `prev` here?

Reply:
Thanks! I have pushed the commit to use `prev` here.